src/custom/utils/index.ts

```diff
@@ -79,3 +79,7 @@ export function getEtherscanLink(chainId: ChainId, data: string, type: BlockExpl
     return getEtherscanUrl(chainId, data, type)
   }
 }
+
+export function shortenOrderId(orderId: string, chars = 4, length = 114): string {
```

Comment:
magic number, actually I have this const in one PR, maybe we just add it to our constants and then we both use it 
https://github.com/gnosis/gp-swap-ui/pull/147

anyways, why length is given instead of using orderId.length?
Also, this is not how we shorten orderIds. I believe we don't show the 0x and we show 8 characters.

Reply:
As the description states and the `2` indicates, there is no 0x, will make 8 and use the const